spec/models/game_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe Game, type: :model do
+  let(:game) { create(:game_with_matches) }
+
+  describe ".name" do
+    it "is a string" do
+      expect(game.name).to be_a(String)
+    end
+  end
+
+  describe ".image" do
+    it "is present" do
+      expect(game.image).to be_present
+    end
+  end
+
+  describe "associations" do
+    it "includes matches" do
+      expect(game.match).not_to be_empty
+    end
+  end
+
+  describe "validations" do
+    context ".name" do
+      subject { built_game }
+      before { create(:game, name: "our_game") }
+
+      let(:built_game) { build(:game, name: "") }
+      it "is present" do
```

Comment:
the test name is bad. it should be something like "cannot be blank"
and the `before` block above doesn't matter here